vault/logical_system.go

```diff
@@ -4008,6 +4008,8 @@ func (b *SystemBackend) pathInternalOpenAPI(ctx context.Context, req *logical.Re
 	// be received from plugin backends.
 	doc := framework.NewOASDocument()
 
+	genericPaths := d.Get("generic_paths").(bool)
```

Comment:
What happens if `generic_paths` isn't passed as a parameter? Does this panic on the type assertion?